### DISCUSSION TITLE
2.6.6 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#2550] [讨论]
· MyHint 控件支持触发自定义事件（EventType 与 EventData）
· 优化加载联网主页失败时的报错处理
· 更新 Mod 数据库
· 重新更新了赞助者列表，如果之前遗漏了非常抱歉！
· 修复了以下 3 个 Bug
　　[重要] 无法使用镜像源下载版本 JSON
　　资源搜索页面重置条件未重置来源（#2537，@g-mph）
　　特定情况下 Mod 名称会显示无用的信息

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.6'>2.6.6</a>.</em>

**Comment by kjjjjjjjjjjjj** [comment]
I love pcl
